scss/components/templates/_page-layout---global.scss

```diff
@@ -0,0 +1,19 @@
+.page-layout {
+  max-width: $site-max-width;
+  margin: 0 auto;
+
+  @include grid-media($grid-xs) {
+    @include grid-container($grid-xs);
```

Comment:
I don't think we need to set the class as a container for each breakpoint. I believe you can move this outside of the grid-media wrapper.